Source/CesiumRuntime/Public/CesiumGeoreference.h

```diff
@@ -102,6 +102,7 @@ class CESIUMRUNTIME_API ACesiumGeoreference : public AActor {
    */
   UPROPERTY(
       EditAnywhere,
+      BlueprintReadOnly,
```

Comment:
It's a little cleaner to:
1. Move this property into the `private` section.
2. Add `Meta = (AllowPrivateAccess)` so that it can be accessed even though it's private.
3. Add `BlueprintReadWrite`, `BlueprintGetter`, and `BlueprintSetter` attributes.
4. Make the getter and setter functions public.

This way C++ can use the getter/setter and Blueprints can access it as a normal property.

Explanation for most of this can be found in this document I wrote awhile back:
https://github.com/CesiumGS/cesium-unreal/blob/ue5-main/Documentation/developing-uobjects.md